phpBB/phpbb/console/command/searchindex/create.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\console\command\searchindex;
+
+use phpbb\config\config;
+use phpbb\console\command\command;
+use phpbb\language\language;
+use phpbb\log\log;
+use phpbb\post\post_helper;
+use phpbb\search\exception\no_search_backend_found_exception;
+use phpbb\search\search_backend_factory;
+use phpbb\search\state_helper;
+use phpbb\user;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class create extends command
+{
+	/** @var config */
+	protected $config;
+
+	/** @var language */
+	protected $language;
+
+	/** @var log */
+	protected $log;
+
+	/** @var post_helper */
+	protected $post_helper;
+
+	/** @var search_backend_factory */
+	protected $search_backend_factory;
+
+	/** @var state_helper */
+	protected $state_helper;
+
+	/**
+	 * Construct method
+	 *
+	 * @param config                     $config
```

Comment:
Use tabs for indentation / alignment

Reply:
This was done automaticaly with the phpstorm config from the documentation